test/e2e/adminapi.go

```diff
@@ -64,7 +66,8 @@ func adminRequest(ctx context.Context, method, path string, params url.Values, i
 		return nil, err
 	}
 	defer func() {
-		_, _ = resp.Body.Read(nil)
+		//_, _ = resp.Body.Read(nil)
```

Comment:
nit: delete the commented out code please

Reply:
Done, .. sorry about that .. I did not realize that I left something commented. Pushed the change.